### DISCUSSION TITLE
Can't apply for Github student developer pack

### DISCUSSION THREAD

**Discussion by GmrezaH** [community/community#55366] [GitHub Education]
### Hi everyone,

I selected my school-issued email and my school is listed in github, but when I click "continue" to be prompted to share my location (and I allowed it) the "continue" button gets pale and nothing happens (no errors or messages).
I've tried couple of times but it's the same situation every time.
what should I do?

**Comment by jge162** [comment]
Greyed out usually means invalid character maybe in the text fill box. You didn’t copy paste the email in right? Try going from a desktop instead.

**Reply by GmrezaH** [reply]
I don't think that would be the case, cause I registered my email in my account and in that page I just choose it. 
and if there was a problem I wouldn't be prompted to allow my location; right?

**Comment by ryanlua** [comment]
Hi @GmrezaH,

Can you provide a screenshot of the specific page (with private info removed of course) along with the URL so I can see what exactly is wrong and where. 

Additionally have you tried using a different browser as sometimes extensions and cookies might affect how GitHub behaves.

**Reply by GmrezaH** [reply]
I used another browser and it worked.
thanks

**Reply by ryanlua** [reply]
No problem and glad to help you. You can mark my comment as the answer to pin it and label this thread as answered. 

Hope you enjoy the Developer Pack 😄
